.github/problem-matchers/depmod.json

```diff
@@ -0,0 +1,17 @@
+{
+    "problemMatcher": [
+        {
+            "__comment_owner": "match depmod warnings/errors",
+            "owner": "depmod",
+            "pattern": [
+                {
+                    "__comment_regexp1": "depmod: ERROR: Cycle detected: crypto -> crypto_algapi -> crypto",
+                    "__comment_regexp2": "depmod: ERROR: Found 2 modules in dependency cycles!",
+                    "regexp": "^depmod:\\s+(FATAL|ERROR|WARNING|NOTICE|INFO|DEBUG):\\s+(.*)$",
```

Comment:
I though "severity" only knows about the literal strings "error" and "warning"? (Does this actually work for the other strings?)

Reply:
Well, that is what the GitHub Actions documents say BUT the [setup-node](https://github.com/actions/setup-node/tree/main/.github) matchers also match on `info`... so I have no idea? Worst case, we roll the dice with this, it doesn't work, then we change the matching.

Reply:
hmm! interesting. If it's an arbitrary string, we might be able to do really weird stuff with it. :) I wonder how it colorizes non-warn and non-error? I guess we'll find out. ;)